### PR TITLE
Update quickstart,leave section and delete section for user guide

### LINKED ISSUE #156: UG: Update quickstart in user guide
- Add image 
- Specify product version
- Provide java installation instructions
- Provide instructions on how to check java version (for Mac Users need to use open JDK Zulu)
- Update example

### LINKED ISSUE #169: [PE-D][Tester B] Delete command in UG not consistent
The UG mentions one of two cases, so I read it as I can delete an employee either by specifying an ID or specifying a name. The latter does not work and the format of the command also does not state that I could delete via a name.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: KIANSEONG/ped#1

### PR DESCRIPTION
**Pull request by chao890**
Resolves #156
For quickstart:
- Add image
- Specify product version
- Provide java installation instructions
- Provide instructions on how to check java version (for Mac Users need to use open JDK Zulu)
- Update example

For leave section:
- Make style and tone of the document more friendly (make it more personal)
- Add descriptions and use examples for different segments of leave management
- Add screenshots of the UI for `delete-leave` and `add-leave`
- Add more clarity and depth to the features, including tables for parameters which shows whether they are optional or not.
- Fix hyperlinks for leave section 

Also resolves #169 from PED

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -68,25 +68,32 @@ Coydir is a desktop app to manage the employee details within a company, optimiz
 ## Quick start
 
 1. Ensure you have Java `11` or above installed in your Computer.
+    1. If you have installed Java before, check that you have the right version,  which is Java `11`.
+        - If you are using Windows, open up command prompt and type `java -version` and enter. 
+        - If you are using Mac, open up terminal and type `java -version` and enter.
```
Comment (kevinchangjk) [inline docs/UserGuide.md:73]: Need to add the note for Mac users also, on using the OpenJDK Zulu build.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -68,25 +68,32 @@ Coydir is a desktop app to manage the employee details within a company, optimiz
 ## Quick start
 
 1. Ensure you have Java `11` or above installed in your Computer.
+    1. If you have installed Java before, check that you have the right version,  which is Java `11`.
+        - If you are using Windows, open up command prompt and type `java -version` and enter. 
+        - If you are using Mac, open up terminal and type `java -version` and enter.   
+    2. If you do not have Java `11`, you can install it from [here](https://www.oracle.com/java/technologies/downloads/).     
+    <br />
+        
+2. Download the latest Coydir v1.3.0 jar file from [here](https://github.com/AY2223S1-CS2103T-T15-1/tp/releases).
```
Comment (kevinchangjk) [inline docs/UserGuide.md:77]: It'll be `v.1.4.0` or maybe `v1.4.1` later on, but actually I think it's fine to just say "latest version".

File: docs/UserGuide.md [hunk 3]
```diff
@@ -260,42 +267,63 @@ However, with Coydir, we can help you to ensure your **company operations run sm
...
 Here is how we support you in leave management.
 
-#### Setting total leaves for an employee
+#### Controlling total leave for an employee
 
-**[Insert description of using `add` and `edit` to set `totalLeave`]**
+You can set the total leave available for an **incoming new** employee when adding them to the database. This can be done by including the optional leave field, `l/`, when using the `add` command. 
 
-<h4 id="is-this-employee-on-leave-today">Is this employee on leave today?</h4>
+Example: `add n/Yi Long Ma j/Chief Operating Officer d/General Management l/20` adds an employee, and specifies his **total leave** to be **20**.
```
Comment (kevinchangjk) [inline docs/UserGuide.md:274]: Good to see my man Yi Long.

File: docs/UserGuide.md [hunk 4]
```diff
@@ -260,42 +267,63 @@ However, with Coydir, we can help you to ensure your **company operations run sm
...
 
-**[Insert description of how Coydir enables live tracking of `isOnLeave`]**
+You can also edit the total leave available for an ****existing** employee by using the `edit` command, and editing the `l/` tag for an employee.
```
Comment (kevinchangjk) [inline docs/UserGuide.md:276]: Extra ** here I think.

**Review by kevinchangjk — APPROVED** [review]
LGTM.

**Review by kevinchangjk — CHANGES_REQUESTED** [review]
Wait I forgot need some changes.

**Review by prit3010 — CHANGES_REQUESTED** [review]
I think need to the new function cause its now `add-leave` instead of `addleave` in the summary table?

File: docs/UserGuide.md (hunk 2 above)
Comment (chao890) [reply docs/UserGuide.md:77]: yes

**Review by prit3010 — COMMENTED** [review]
LTGM

**Review by prit3010 — APPROVED** [review]

**Review by kevinchangjk — APPROVED** [review]
Nicely.

**Pull request merged**